pyaedt/rpc/rpyc_services.py

```diff
@@ -30,6 +31,33 @@
 from pyaedt import Mechanical
 
 
+def check_port(port):
+    """Check for an available port on the machine starting from input port.
+
+     Parameters
+    ----------
+    port : int
+        Ports to search.
+
+    Returns
+    -------
+    int
+        Next Port available.
+    """
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    check = False
+    while not check:
+        try:
+            s.bind((socket.getfqdn(), port))
+            check = True
+        except socket.error as e:
+            port += 1
+            if port > 29999:
```

Comment:
@maxcapodi78 Please add a small comment to explain why this limit "29999". I might be missing something obvious.

Reply:
Done